packages/auth/amplify_auth_cognito_dart/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.10.6
+
+### Fixes
+- fix(aft): Small fixes ([#3838](https://github.com/aws-amplify/amplify-flutter/pull/3838))
```

Comment:
*todo:* Let's exclude this. It modified a file in amplify_auth_cognito_dart, but it was only in the /tool dir. It shouldn't be user facing.